recipes/openssl/3.x.x/conanfile.py

```diff
@@ -712,7 +712,7 @@ def _module_file_rel_path(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "OpenSSL")
-        self.cpp_info.set_property("cmake_target_name", "OpenSSL")
+        self.cpp_info.set_property("cmake_target_name", "OpenSSL::OpenSSL")
```

Comment:
doesn't exist in official FindOpenSSLcmake

Reply:
```suggestion
```

Reply:
Is `cmake_find_mode=both` defining both `OpenSSL` AND `OpenSSL:OpenSSL` targets? 
Wasn't the idea of those change to be more explicit?

Reply:
Sorry, I should have made that in another comment, It's two separate things, in one side removing that target that is not necessary and on the other forcing CMakeDeps to create the FindOpenSSL file

Reply:
ah nice, interesting feature. so libcurl and sqlite3 also deserve `cmake_find_mode`.